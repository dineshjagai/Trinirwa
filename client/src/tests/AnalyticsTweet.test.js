import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react'
import DialogAnalytics from '../components/AnalyticsTweet';

test('Test AnalyticsTweet snapshot', () => {
  const component = renderer.create(<DialogAnalytics tweetId={1}/>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

 test('checks that the labels are there', () => {
      const { getByText } = render(<DialogAnalytics tweetId={1} />);
      getByText('Analytics');
  });
