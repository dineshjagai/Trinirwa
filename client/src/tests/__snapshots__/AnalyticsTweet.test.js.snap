// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test AnalyticsTweet snapshot 1`] = `
<div>
  <div>
    <button
      className="MuiButtonBase-root MuiIconButton-root"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "borderRadius": "50%",
          "padding": "3",
        }
      }
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z"
          />
        </svg>
      </span>
    </button>
    <span
      id="analytics_btn"
      style={
        Object {
          "color": "#0C8367",
        }
      }
    >
      Analytics
    </span>
  </div>
</div>
`;
